src/common/constants/dev-config.json

```diff
@@ -24,11 +24,11 @@
   "mlarReleaseYear": "2021",
   "maintenanceMode": false,
   "filingAnnouncement": {
-    "heading": "Collection of 2018 HMDA Data Will End",
+    "heading": "Collection of 2019 HMDA Data Will End",
     "type": "warning",
-    "message": "Resubmissions of 2018 HMDA data will not be accepted after December 31st, 2021.",
+    "message": "Resubmissions of 2019 HMDA data will not be accepted after December 31st, 2022.",
     "link": {
-      "url": "https://ffiec.cfpb.gov/documentation/2021/data-collection-timelines",
+      "url": "https://ffiec.cfpb.gov/documentation/2022/data-collection-timelines",
       "text": "View the HMDA data collection timelines."
     },
     "endDate": "12/31/2021"
```

Comment:
Sorry, I forgot what the current year is 🙈
```suggestion
    "endDate": "12/31/2022"
```

Reply:
`endDate` has been updated. 

Banner before user sign in:

<img width="1065" alt="image" src="https://user-images.githubusercontent.com/60801873/202497837-4b2e0187-575f-4f12-b0dc-98cfe0d79c22.png">

Banner is still present after user sign in.